x-pack/plugins/ml/public/settings/filter_lists/components/add_item_popover/add_item_popover.js

```diff
@@ -97,7 +108,10 @@ export class AddItemPopover extends Component {
         >
           <EuiForm>
             <EuiFormRow
-              label="Items"
+              label={intl.formatMessage({
```

Comment:
please replace intl to **FormattedMessage** and remove **injectI18n** from this file